addons/hr_holidays/models/hr_leave.py

```diff
@@ -1137,7 +1137,7 @@ def action_approve(self, check_state=True):
 
         # Do not check the state in case we are redirected from the dashboard
         if check_state and any(holiday.state != 'confirm' for holiday in self):
-            raise UserError(_('Time off request must be confirmed ("To Approve") in order to approve it.'))
+            raise UserError(_('You can only manage time off requests that are in state “To approve”'))
```

Comment:
manage is a bit more general word. It's confusing.. we are confirming the time off, so let's call it so